bokeh/palettes.py

```diff
@@ -1381,4 +2227,4 @@ def gray(self, n):
 _mod.__doc__ = __doc__
 _mod.__all__ = dir(_mod)
 _sys.modules['bokeh.palettes'] = _mod
-del _autoprop, _mod, _sys, _types
+del _mod, _sys, _types
```

Comment:
I was about to say could remove `Dict` and `List` here but I'm not actually sure we need to remove anything, since the actual `sys.modules` entry is not this module but the `_PalettesModule` instance

Reply:
That make's sense. I didn't consider this issue at all when adding `typing`'s imports.